setup.cfg

```diff
@@ -31,7 +31,7 @@ package_dir =
     asdf.reference_files = asdf-standard/reference_files
 include_package_data = True
 install_requires =
-    semantic_version>=2.3.1,<=2.6.0
+    semantic_version>=2.8
```

Comment:
Could this be relaxed to 2.7, when `SimpleSpec` was introduced?  Or were functional changes made in 2.8 that we need?

Reply:
2.7 was a short lived release, looked like it wasn't stable.

Reply:
Ah ok.  👍